front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -361,6 +366,32 @@ export default function LabsTranscriptsIndex({
     }
   };
 
+  const handleDisconnectProvider = async () => {
+    if (!transcriptsConfiguration) {
+      return;
+    }
+
+    const response = await fetch(
+      `/api/w/${owner.sId}/labs/transcripts/${transcriptsConfiguration.id}`,
+      {
+        method: "DELETE",
+      }
+    );
+
+    if (!response.ok) {
+      sendNotification({
+        type: "error",
+        title: "Failed to disconnect provider",
+        description:
+          "Could not disconnect from your transcripts provider. Please try again.",
+      });
+    } else {
+      window.location.reload();
```

Comment:
😱 Why not using `mutateTranscriptsConfiguration`?

Reply:
Used it but nothing happens :')

Reply:
Then it means that your endpoint should return an empty array in this case 😛